vars/RedHat_8.yml

```diff
@@ -0,0 +1,26 @@
+# roles/bind/vars/RedHat.yml
+---
+
+bind_packages:
+  - python3-netaddr
```

Comment:
Rather than creating a whole new file, with just this one change, I would suggest changing this one line to:

  - "{{ ( ansible_distribution_major_version == '8' ) | ternary( 'python3-netaddr', 'python-netaddr' ) }}"

Just to try and adhere to the DRY principles closer....

Reply:
Good idea, updated PR.